modules/apps/client-extension/client-extension-type-impl/src/main/java/com/liferay/client/extension/type/internal/factory/GlobalCSSCETImplFactoryImpl.java

```diff
@@ -62,7 +62,9 @@ public void validate(
 
 		if (!Validator.isUrl(newGlobalCSSCET.getURL())) {
 			throw new ClientExtensionEntryTypeSettingsException(
-				"please-enter-a-valid-url");
+				"Please enter a valid URL",
```

Comment:
All instance of invalid URLs should follow the same pattern: `"Invalid URL: " + url,`